src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/BasePageObject.java

```diff
@@ -321,9 +321,9 @@ public void getUrl(Page page, String queryString) {
   public void refreshPage() {
     try {
       driver.navigate().refresh();
-      PageObjectLogging.log("refreshPage", "page refreshed", true);
+      Log.log("refreshPage", "page refreshed", true);
     } catch (TimeoutException e) {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)